dts/arm/st/stm32f4.dtsi

```diff
@@ -145,6 +145,19 @@
 			status = "disabled";
 			label = "SPI_2";
 		};
+
+		usbotg_fs: usb@50000000 {
+			compatible = "st,stm32-otgfs";
```

Comment:
To be consistant with Linux , should be "st,stm32-fsotg". Though, Linux is not consistant internally and we can see OTGFS later on for clocks.

So let it keep that way for Zephyr.  This is fine

Reply:
Ack